entrypoint.sh

```diff
@@ -29,27 +29,38 @@ if echo ${INPUT_VERBOSITY} | grep -Eq '[^0-9]'; then
 fi
 echo "Using verbosity level: ${INPUT_VERBOSITY}"
 
-i=0
-while [[ $i -lt $INPUT_VERBOSITY ]]
-do
+for (( i=0; i<${INPUT_VERBOSITY}; i++ )); do
```

Comment:
Haha, I tried *exactly* this locally and it complained about syntax errors

Reply:
Not for me :) Also not if you run tests in the Docker image :) Does it still not work for you? What version of bash do you have?

Reply:
```
./entrypoint.sh: line 32: ((: i<: syntax error: operand expected (error token is "<")
```
```
$ /usr/bin/env bash --version
GNU bash, version 5.0.17(1)-release (x86_64-pc-linux-gnu)
```

Reply:
Ah.. huh.. yeah. It seems v5 deprecated the C-style for loop... hmm

Reply:
I'll revert to the the more chunky while loop 👍

Reply:
I was running v4.4...